src/ripple/app/tx/impl/Payment.cpp

```diff
@@ -160,6 +161,55 @@ class Payment
                 "No ripple direct specified for XRP to XRP.";
             return temBAD_SEND_XRP_NO_DIRECT;
         }
+        if (bDeliverMin)
+        {
+            auto const deliverMin = mTxn.getFieldAmount (sfDeliverMin);
```

Comment:
Vinnie made a comment on slack that asked about partial payment. I think we need to check that partial payment is set or deliver min doesn't make sense.